src/repl-frontend/scala/tools/nsc/interpreter/shell/InteractiveReader.scala

```diff
@@ -34,6 +32,8 @@ trait InteractiveReader {
     // hack necessary for OSX jvm suspension because read calls are not restarted after SIGTSTP
     if (scala.util.Properties.isMac) restartSysCalls(readOneLine(prompt), reset())
     else readOneLine(prompt)
+
+  def initCompletion(completion: Completion): Unit = {}
```

Comment:
I think I was thinking that I prefer `def f(): Unit = ()` style instead of braces. Are braces the new tabs?